lxd/db/generate/db/stmt.go

```diff
@@ -260,6 +225,61 @@ func (s *Stmt) objects(buf *file.Buffer) error {
 	return nil
 }
 
+// objectsBy parses the variable declaration produced by the 'objects' function, and appends a WHERE clause to its SQL
+// string using the objects-by-<FIELD> field suffixes, and then creates a new variable declaration.
+// Strictly, it will look for variables of the form 'var = <entity>Objects = <database>.RegisterStmt(`SQL String`)'.
```

Comment:
Did you mean `var <entity>Objects = <database>.RegisterStmt(`SQL String`)` here (comparing to the commit comment itself)?

Reply:
Also does this var need to be exported? If so would be worth indicating that. But im not sure if the DB generator worries about such things :)

Reply:
Oops, good catch! 

And no, the var doesn't need to be exported, even if it's in a different package.

Reply:
Fixed the comment.